lib/matplotlib/streamplot.py

```diff
@@ -575,7 +575,7 @@ def _integrate_rk12(x0, y0, dmap, f, maxlength):
         dx2 = ds * 0.5 * (k1x + k2x)
         dy2 = ds * 0.5 * (k1y + k2y)
 
-        nx, ny = dmap.grid.shape
+        ny, nx = dmap.grid.shape
         # Error is normalized to the axes coordinates
         error = np.hypot((dx2 - dx1) / (nx - 1), (dy2 - dy1) / (ny - 1))
```

Comment:
We could alternatively do
```suggestion
        error = np.hypot((dx2 - dx1) / (dmap.grid.nx - 1),
                         (dy2 - dy1) / (dmap.grid.ny - 1))
```
which avoids the error-prone shape order.

Reply:
.. a proper test should have caught the bug though.  I think its worth fixing the test.

Reply:
hello @jklymak! I'm back in town. What would be required to move forward? Doing a PR for the test?

Reply:
Just update the test in this PR.